src/Jenssegers/Mongodb/Query/Builder.php

```diff
@@ -1018,6 +1029,20 @@ protected function compileWhereRaw($where)
     {
         return $where['sql'];
     }
+    
+    /**
+     * Convert to MongoDB\BSON\UTCDateTime if $value is a DateTime object.
+     *
+     * @param  mixed   $value
+     * @return mixed
+     */
+    protected function dateTimeConvertion($value)
+    {
+        if ($value instanceof DateTime) {
+            return new UTCDateTime($value->getTimestamp() * 1000);
```

Comment:
Isn't the if conditional check redundant? You are already checking if the value is an instance of DateTime before calling the dateTimeConversion function.